### ISSUE TITLE
❌  🛟 ♿ Ally Accessibility option to Stop Animations doesn't apply to autoplay videos

### ISSUE THREAD

**Issue by rw2wpgfrnv-beep** [elementor/elementor#34582]
### Description

The Ally Accessibility tool offers the option to stop animations for people sensitive to motion and light. However, this function ignores animations that are run as background videos.

### Steps to reproduce

1. Install Ally Accessibility Plugin
2. Press Button „stop animations“

### Expected behavior

All animations should be stoped (including videos)

### Elementor System Info

```txt
Wordpress and Elementor Pro
```

### Agreement

- [ ] I confirm I have read and followed all the guidelines and instructions outlined in the Elementor Bug Report form.
- [x] I agree that my issue may be closed without further action if it doesn't meet all the requirements outlined in the Elementor Bug Report form.

**Comment by mbaierl** [comment]
Ideally no extra plugin is needed, Elementor has to be a11y compatible by default.

**Comment by nicholaszein** [comment]
This issue does not comply with our [Contribution Guidelines for submitting Bug Reports](https://elemn.to/gh-contributing-bug-reports). It is missing required information such as the non-optional Elementor System Info.

For this reason we cannot continue the validation process.

This repository is also for the Elementor Editor, not for other products.

For this reason I'm closing this issue.

Please use the Ally WordPress Forum, or reach out to Elementor Support through your Elementor Account if you have an Ally active license subscription.

Regards
